.github/workflows/ci.yml

```diff
@@ -131,17 +136,17 @@ jobs:
       id-token: write
       pull-requests: write
     env:
-      API_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/api-perf'
+      API_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/api'
       API_TAG: ${{ matrix.sha }}
-      WEB_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/web-perf'
+      WEB_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/web'
       WEB_TAG: ${{ matrix.sha }}
-      ELIXIR_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/elixir-perf'
+      ELIXIR_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/elixir'
       ELIXIR_TAG: ${{ matrix.sha }}
-      GATEWAY_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/gateway-perf'
+      GATEWAY_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/perf/gateway'
       GATEWAY_TAG: ${{ matrix.sha }}
-      CLIENT_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/client-perf'
+      CLIENT_IMAGE: 'us-east1-docker.pkg.dev/firezone-staging/firezone/perf/client'
```

Comment:
Is the `perf` is just non-debug/production one? If yes, maybe let's just not have a prefix for production-ready released and have it for `debug` ones?

Reply:
`perf` is `debug` stage image + `release` binaries. Subtle distinction that I tried to clear up.

I thought it would be nice to have readily-available performance images we can pull locally to do performance testing.

Reply:
I'll add more comments since @ReactorScram asked the same question